my_enumerable.rb

```diff
@@ -0,0 +1,19 @@
+module MYENUMERABLE
+  def all?()
+    result = true
+    each { |e| result = false unless yield e }
+    print result
```

Comment:
## OPTIONAL

- [ ] If you use 'p' instead of 'print' it will show the results as a column as follows:
![Screen Shot 2022-02-24 at 17 45 49](https://user-images.githubusercontent.com/67211919/155626423-7949a141-198e-4558-9323-7f0fc85da9c9.png)

It has a better look but that's up to you. 😉

Reply:
Awesome, @eri8-9, that looks so much better indeed! 👍